test/integration/voldemort/performance/benchmark/Benchmark.java

```diff
@@ -642,6 +651,18 @@ public static void main(String args[]) throws IOException {
               .withRequiredArg()
               .describedAs(LOCAL_SERVER_PROPERTIES)
               .ofType(String.class);
+        parser.accepts(WORKLOAD_CLASS, "workload class (full name)")
+              .withRequiredArg()
+              .describedAs(WORKLOAD_CLASS)
+              .ofType(String.class);
```

Comment:
I like your approach. But I believe using CLASS name in the option is misleading and sort of suggests reflection and other.

WORKLOAD_TYPE may be a better name in my opinion.